verde/coordinates.py

```diff
@@ -702,3 +729,41 @@ def block_split(coordinates, spacing, adjust="spacing", region=None):
     tree = kdtree(block_coords)
     labels = tree.query(np.transpose(n_1d_arrays(coordinates, 2)))[1]
     return block_coords, labels
+
+
+def _latlon_continuity(coordinates, region):
+    """
+    Modify longitudinal geographic coordinates to ensure continuity around the globe.
+    """
+    w, e, s, n = region[:]
+    longitude, latitude = coordinates[:2]
+    # Check if region is defined all around the globe
+    all_globe = np.allclose(abs(e - w), 360)
+    # Move coordinates to [0, 360)
+    w = w % 360
+    e = e % 360
+    longitude = longitude % 360
+    # Move west=0 and east=360 if region longitudes goes all around the globe
+    if all_globe:
+        w, e = 0, 360
+    # Check if the [-180, 180) interval is better suited
+    if w > e:
+        e = ((e + 180) % 360) - 180
+        w = ((w + 180) % 360) - 180
+        longitude = ((longitude + 180) % 360) - 180
+    region = [w, e, s, n]
+    coordinates = [longitude, latitude]
+    return coordinates, region
+
+
+def _check_geographic_coordinates(coordinates):
+    "Check if geographic coordinates are within accepted degrees intervals"
+    longitude, latitude = coordinates[:2]
+    if (longitude > 360).any() or (longitude < -180).any():
```

Comment:
Does `.any()` work with `pandas.Series`? If not, the maybe calling `np.any` might be better.